### PR TITLE
add total today

### PR DESCRIPTION
**Pull request by henriwasd**
Add Function for sum total the tasks of today

### CODE REVIEW

**Comment by Ellpeck** [comment]
Hi!! Thanks so much for this pull request :)

Please update it so that it doesn't change the formatting of the entire file by removing the semicolons.

**Comment by henriwasd** [comment]
Adjustment done

**Review by Ellpeck — CHANGES_REQUESTED** [review]
Thanks!! Just a few more notes now that the diff is a bit clearer to see :)

File: src/tracker.ts [hunk 1]
```diff
@@ -100,6 +100,9 @@ export function displayTracker(tracker: Tracker, element: HTMLElement, getFile:
     let totalDiv = timer.createEl("div", { cls: "simple-time-tracker-timer" });
     let total = totalDiv.createEl("span", { cls: "simple-time-tracker-timer-time", text: "0s" });
     totalDiv.createEl("span", { text: "Total" });
+    let totalTodayDiv = timer.createEl("div", { cls: "simple-time-tracker-timer" })
```
[the remark this answers is not in the document]
Comment (henriwasd) [reply src/tracker.ts:103]: Yes, I tested it on mobile, I even use it on mobile too, so it's working fine, but I'm going to implement the function in the options too

**Comment by henriwasd** [comment]
I added both today's total functions to the API, because I saw that it had both total time functions

**Review by Ellpeck — APPROVED** [review]
Thanks so much, this looks great!!

**Pull request merged**